src/engine/BMInterface.php

```diff
@@ -1385,18 +1337,19 @@ protected function save_player_with_initiative($game) {
                      'AND player_id = :player_id;';
             $statement = self::$conn->prepare($query);
             $statement->execute(array(':game_id' => $game->gameId,
-                                      ':player_id' => $game->playerIdArray[$game->playerWithInitiativeIdx]));
+                                      ':player_id' => $game->playerArray[$game->playerWithInitiativeIdx]
+                                                           ->playerId));
```

Comment:
Nit: i think two-line variable accesses like this are a little suspect.  Would our linewrapping allow us to format this block as something like:
```php
$statement->execute(
    array(':game_id' => $game->gameId,
          ':player_id' => $game->playerArray[$game->playerWithInitiativeIdx]->playerId));
```